aspnetcore-webapi/RateLimitingDotNET8/RateLimitingDotNET8/RateLimiters.cs

```diff
@@ -0,0 +1,149 @@
+﻿using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.RateLimiting;
+using Microsoft.Extensions.Primitives;
+using System.Threading.RateLimiting;
+
+namespace RateLimitingDotNET8;
+
+public static class RateLimiters
+{
+    public static void FixedRateLimiter(WebApplicationBuilder builder)
+    {
+        var fixedOptions = builder.Configuration.GetSection(FixedOptions.Fixed).Get<FixedOptions>();
```

Comment:
The above does not look to me to be the correct way to read the configuration from the file since you have already loaded the configuration in the FixedOptions class in the Extension method.

Once the configuration is loaded in the class, it can be used from IOptions<ClassName> IOptionsSnapshot<ClassName>